### ISSUE TITLE
MetricSecondaryStructure incorrect dtype when argument simplified is False

### ISSUE THREAD

**Issue by hawk31** [Acellera/htmd#447]
Minimal replication in HTMD 1.9.7

```python 
from htmd.molecule.molecule import Molecule
from htmd.projections.metricsecondarystructure import MetricSecondaryStructure


mol = Molecule('3PTB')
mol.renumberResidues()
mol.filter('protein')

mss = MetricSecondaryStructure(simplified=False)
mss.project(mol)
```

```pytb
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-1-b8dcc22d556e> in <module>()
      8 
      9 mss = MetricSecondaryStructure(simplified=False)
---> 10 mss.project(mol)

/shared/jose/htmd/htmd/projections/metricsecondarystructure.py in project(self, mol)
    130             data[data == 'E'] = 1
    131             data[data == 'C'] = 0
--> 132             data = np.array(data, dtype=np.int32)
    133         #data[:, np.logical_not(protein_indices)] = 'NA'
    134 

ValueError: invalid literal for int() with base 10: ' '
```

Esentially the dtype is fine for `simplified=True` (since you are assigning 0, 1 and 2s) but not for the general case.

**Comment by stefdoerr** [comment]
Oh yeah right. You should use `simplified=False` with `integer=False`.
I don't have a mapping for non-simplified to integer although I can make one

**Comment by hawk31** [comment]
Is there any reason for the integer mapping? I mean, the letter typing is just fine and it just simplifies the dtyping.

**Comment by stefdoerr** [comment]
convention and memory efficiency

**Comment by stefdoerr** [comment]
Fixed 8f3dad320a960c1e51b42a5b4ff3814cefcee97b
